### PR TITLE
doc(open meetings): add new open meeting for edc weekly

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description
This PR adds a new open meeting to our open meetings page. Requester for this change is @stefan-ettl  and @benwestphal

image

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

image

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by stephanbcbauer** [comment]
Can someone support here? @FaGru3n , @SebastianBezold ? I ran the dependency check locally, but I think it's a different result? Thanks for support

**Comment by SebastianBezold** [comment]
> Can someone support here? @FaGru3n , @SebastianBezold ? I ran the dependency check locally, but I think it's a different result? Thanks for support

I had the check fail on a different PR too, but this was due to a technical issue with ClearlyDefined. Re-run worked without issue. Seems to be the case here as well

**Review by SebastianBezold — APPROVED** [review]

**Review by FaGru3n — APPROVED** [review]
LGTM

**Comment by stephanbcbauer** [comment]
@stefan-ettl, @benwestphal could you please review from content perspective? Thanks

**Review by FaGru3n — DISMISSED** [review]

**Review by stefan-ettl — APPROVED** [review]

**Comment by stephanbcbauer** [comment]
[heart]         Stephan Bauer reacted to your message:
________________________________

File: community/open-meetings.mdx [hunk 1]
```diff
@@ -24,7 +32,7 @@ These are dedicated sync meetings for specific products, as well as open plannin
 />
 
 <MeetingInfo title="Committer Meeting"
-             schedule="Every third Friday of the month effective 16. Feb 2024 until 31. Dec 2024 from 02:05 pm to 03:00 pm CET"
+             schedule="Eevery 2 weeks on Friday effective 16. Feb 2024 until 31. Dec 2024 from 02:05 pm to 03:00 pm CET"
```
Comment (carslen) [inline community/open-meetings.mdx:35]:
```suggestion
             schedule="Every 2 weeks on Friday effective 16. Feb 2024 until 31. Dec 2024 from 02:05 pm to 03:00 pm CET"
```

Comment (carslen) [reply community/open-meetings.mdx:35]: Fix typo

**Review by carslen — CHANGES_REQUESTED** [review]

**Review by FaGru3n — APPROVED** [review]

**Review by carslen — APPROVED** [review]
👍

**Pull request merged**